src/early_multicellular_stage/editor/CellBodyPlanEditorComponent.Callbacks.cs

```diff
@@ -8,12 +8,16 @@ public partial class CellBodyPlanEditorComponent
     private void OnCellAdded(HexWithData<CellTemplate> hexWithData)
     {
         cellDataDirty = true;
+
+        UpdateReproductionOrderList();
```

Comment:
This should get triggered like the other updates with the cell data dirty. I think there's already a ton of systems that get updated based on the cellDataDirty variable.